src/components/card/FLContributorProfileCard.tsx

```diff
@@ -45,11 +48,24 @@ export function FLContributorProfileCard({
                             color: theme.palette.foodloop.main,
                             textDecoration: 'none',
                             cursor: 'pointer',
+                            marginRight: '10px',
```

Comment:
Adding a `marginRight` here is fine, but it seems very "hardcoded". We can add a `gap` property to the `sx` prop in `FLBox` line 43. `gap = "2%"` or `"4%` would do fine!

What gap does is that within the container, each item (each html element) will be spaced out evenly with gap in between

Reply:
okay, fixed. thanks!